maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -85,15 +88,18 @@
 	@Parameter(defaultValue = "${project.build.resources}", readonly = true)
 	private List<org.apache.maven.model.Resource>	resources;
 
-	@Parameter(defaultValue = "${project.build.outputDirectory}")
+	@Parameter(defaultValue = "${project.build.outputDirectory}", readonly = true)
```

Comment:
Why are these back to readonly=true? I thought we expect people to be able to configure this?

Reply:
.. I thought we just wanted to use the flag to say don't add this to the jar automatically, but I can make it configurable if you like.